### DISCUSSION TITLE
Pro as a RNDIS/Ethernet Gadget

### DISCUSSION THREAD

**Discussion by MaciejStawiar** [WLAN-Pi/feedback#34] [General Inquiries] [answered]
Hi All,
For some reason Pro don't want to work as a RNDIS/Ethernet Gadget when connected to an iPad on latest iOS. 
It works well when connected to Windows 11. Also WLANPi NEO works when connected to the same iPad.
I tried to configure this using this tutorial: https://www.youtube.com/watch?v=L8r6kMod7Vw - it worked on normal RPi 4, but don't want to work here.
Any idea/suggestions to this?
I am using this type of connection when want to run profiler from CLI using iPad app, to be sure I captured the frame or change the profiler channel.

**Comment by jolla** [comment]
Which model iPad?

**Reply by MaciejStawiar** [reply]
Air 4 with cellular - A2072

**Comment by MaciejStawiar** [comment, marked as answer]
So I didn't solve this, but found better - wireless solution.
I connect via Bluetooth which also open Ethernet connection and provide IP addr on both sides - then ssh from the iPad to Pro using IP addr provided on the Pro screen.
